src/org/ensembl/healthcheck/testcase/compara/AbstractComparaTestCase.java

```diff
@@ -235,22 +235,33 @@ public DatabaseRegistryEntry getLastComparaReleaseDbre(DatabaseRegistryEntry cur
 	protected final static Pattern EC_DB = Pattern.compile("^([^_]+_)?ensembl_compara_[0-9]+");
 
 	String getComparaDivisionName(DatabaseRegistryEntry compara_dbre) {
+		Connection compara_con = compara_dbre.getConnection();
+		String sql_division = "SELECT meta_value FROM meta WHERE meta_key = 'division'";
+		String division = DBUtils.getRowColumnValue(compara_con, sql_division);
+		if (division == null || division.isEmpty()) {
```

Comment:
I know you're trying to preserve the original indentation, but I think this is difficult to read. Let's return `division` immediately here, and you don't need the `if`

Reply:
But division can be empty (either `meta_value` is empty or _division_ `meta_key` is not there at all), so we still need the conditional, right?

Reply:
What I have seen it is not necessary is to check if division is `null`, as the `meta_value` cannot be `null`.

Reply:
Yes, you still need a conditional, I wasn't clear, but it would be:
```
if there is a division {
    return it
}
```
It woulnd't have to wrap the block below